.gitignore

```diff
@@ -54,3 +54,8 @@ google-upload-credentials.json
 
 # Eclipse
 .settings/
+
+# All secrets should be stored under .configure-files
```

Comment:
After opening this PR, I realized that the wording here may be inappropriate. As a matter of fact, none of the secrets are currently stored in `.configure-files`.

🤔 

I still think there's value in encouraging developers to store all decrypted secrets in a single, _ignored_ location. Along the lines of https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1964.

Reply:
Oguz left a great comment on this here https://github.com/wordpress-mobile/WordPress-Android/pull/14526/files#r619933550